File: 2st/Riyeon/배열에서 문자열 대소문자 변환하기.java
class Solution {
    public String[] solution(String[] strArr) {
        for(int i=0; i<strArr.length; i++){
            if(i%2==0)
                strArr[i] = strArr[i].toLowerCase();
            else
                strArr[i] = strArr[i].toUpperCase();                
        }
        return strArr;
    }
}

File: 3st/Jaeyoung/배열에서 문자열 대소문자 변환하기.js
function solution(strArr) {
    var answer = [];
    for( var i = 0 ; i < strArr.length ; i++ ) {
        i%2 == 0 ? answer[i] = strArr[i].toLowerCase() : answer[i] = strArr[i].toUpperCase()
    }
    return answer;
}